### ISSUE TITLE
[JENKINS-63193] Increase contrast of the disabled plugin uninstall button 

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-15#300]
<p>When the "Uninstall" button on the /<em>pluginManager/installed</em> page is disabled it has too low contrast that can bring in accessibility issues. See screenshot.</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51993/51993_Captura+de+pantalla+2020-07-24+a+las+10.50.18.png" height="282" width="598" style="border: 0px solid black" /></span></p>

<p>Solution:</p>

<ul>
	<li>Remove the CSS rules for the `#plugins tr.has-dependents input.uninstall` selector.</li>
	<li>Bump the opacity of the disabled button from 0.2 to 0.4 or 0.5.</li>
</ul>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/fqueiruga.png" title="fqueiruga's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=fqueiruga">fqueiruga</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-63193" target="_blank">Increase contrast of the disabled plugin uninstall button </a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/benebsiny.svg" title="benebsiny's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=benebsiny">benebsiny</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2020-09-28T11:58:33+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 2025-11-16
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>When the "Uninstall" button on the /<em>pluginManager/installed</em> page is disabled it has too low contrast that can bring in accessibility issues. See screenshot.</p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/51993/51993_Captura+de+pantalla+2020-07-24+a+las+10.50.18.png" height="282" width="598" style="border: 0px solid black" /></span></p>

<p>Solution:</p>

<ul>
	<li>Remove the CSS rules for the `#plugins tr.has-dependents input.uninstall` selector.</li>
	<li>Bump the opacity of the disabled button from 0.2 to 0.4 or 0.5.</li>
</ul>
</pre>
</details>
</details>
<details><summary><i>1 attachment</i></summary>

- [Captura de pantalla 2020-07-24 a las 10.50.18.png](https://issues.jenkins.io/secure/attachment/51993/Captura%20de%20pantalla%202020-07-24%20a%20las%2010.50.18.png)
  > ![Captura de pantalla 2020-07-24 a las 10.50.18.png](https://issues.jenkins.io/secure/attachment/51993/Captura%20de%20pantalla%202020-07-24%20a%20las%2010.50.18.png)
</details>

**Comment by jenkins-infra-bot** [comment]
<i>[Epic: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-15/issues?q=in%3Atitle%20JENKINS-60919%20label%3Aepic">JENKINS-60919</a>]</i>
